.github/actions/prepare-hermes-workspace/action.yml

```diff
@@ -0,0 +1,104 @@
+name: prepare-hermes-workspace
```

Comment:
cherry picked 9c6c63781669bd1d6451c880671a151060220020 for stabilize prepare-hermes-workspace